source/guide_strapi.rst

```diff
@@ -0,0 +1,299 @@
+.. author:: Richard Henkenjohann <richardhenkenjohann@googlemail.com>
+
+.. tag:: lang-nodejs
+.. tag:: web
+.. tag:: backend
+.. tag:: api
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/strapi.svg
+      :align: center
+
+#####
+Strapi
+#####
+
+.. tag_list::
+
+Strapi_ is a open source headless CMS to build APIs. It’s 100% Javascript, fully customizable and developer-first.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We're using :manual:`Node.js <lang-nodejs>` in the stable version 10:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use node 10
+ Using 'Node.js' version: '10'
+ Selected node version 10
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your URL needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Start with a local project
+--------------------------
+
+When Strapi is in production mode (which should be targeted on Uberspace), the Content Type Creator is disabled. This requires us to build the project locally to be able to make changes on the API and deploy it for production on Uberspace.
+
+We will start with creating a new Strapi project. Use yarn to install the Strapi project (locally).
+
+.. code-block:: console
+
+ [local@My-Computer ~]$ yarn create strapi-app my-project --quickstart
```

Comment:
This fails for me with the following output:

```
$ yarn create strapi-app my-project --quickstart
yarn create v1.22.10
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "create-strapi-app@3.2.5" with binaries:
      - create-strapi-app
error Cannot create "/usr/bin/create-strapi-app" due to insufficient permissions.
```

How do you run this as a non-privileged user? Just running `yarn create` as root will also assign ownership of the `my-project` directory to root, which is not what we want here.